.github/workflows/ci.yml

```diff
@@ -51,13 +48,9 @@ jobs:
         run: docker exec -d pgai make test-server
 
       - name: Run tests
-        run: docker exec pgai make test
-
-      - name: Run vectorizer tests
-        run: make test-vectorizer
-
-      - name: Run type check
-        run: make type-check-py
+        run: |
+          docker exec pgai make test
+          docker exec pgai make test-vec
```

Comment:
Wondering if we can parallelize those two jobs rather than running them sequentially

Reply:
I'm backing these changes out. I think we need a more wholistic reworking of the build